code/game/objects/effects/poster_motivational.dm

```diff
@@ -0,0 +1,291 @@
+#define POSTER_MOOD_CAT "poster_mood"
+
+/obj/item/poster/quirk
+	name = "placeholder quirk poster"
+	desc = "Uh oh! You shouldn't have this!"
+	icon_state = "rolled_poster_legit"
+	var/quirk_poster_department = NONE
```

Comment:
DMDoc this. https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md